packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -137,6 +137,7 @@ export function execAsync(
       child.on('close', (code, signal) => {
         if (code === 0 || opts.ignoreNon0Exit) {
           return resolve({
+            // @ts-ignore
```

Comment:
**suggestion (non-blocking):** add comment about why

Can you add a comment here about why this was ignored?